packit_service/worker/reporting.py

```diff
@@ -128,7 +129,8 @@ def report_status_by_comment(
             "| ------------- | ------------ |",
         ] + [f"| [{check}]({url}) | {state.name.upper()} |" for check in check_names]
 
-        self.comment("\n".join(comment_table_rows))
+        table = "\n".join(comment_table_rows)
+        self.comment(table + f"\n### Description\n\n{description}")
```

Comment:
@mfocko I am looking at it now and thinking if we are not doing this too complicated:

* If we want to match status formatting, I would add the description to the table.
    * => Differentiate between `description` and `footer`.
* As a second option, we can change the behaviour a bit:
  * Use table only when chroots are relevant, unfortunately, we don't process all chroots together and I'm not sure how easy is to check this condition.
  * In case of permission errors, it does not make sense to print some chroots at all.

WDYT?

Reply:
I'm looking at it and there's nothing to match tbh
- GitLab
  ```
  IF reporting is allowed THEN
    StatusReporter.report()
    // that tries commit status, if fails, falls back to commit comment
  ELSE
    report by comment (if merge request is present then MR comment else commit comment)
  FI
   ```
- GitHub
  ```
  StatusReporter.report()
  // that tries commit status, if fails, falls back to commit comment
  ```

I'll try to go through the commits if there is any reason we check reporting and don't go by default (I guess the reason would be requesting access which is something that `StatusReporter` doesn't take care of... but we probably could check if it's needed to request permissions and then just call that `report` method

Ad 2nd question, it's weird, because we can get `str` or `List[str]` of check names, but only one description, since there is no problem with that on GitHub, I guess that's something GitLab specific.

Reply:
AI:
- [ ] check the stage as is now
- [ ] clean it up (*aargh*)

Reply:
current state of stage: https://gitlab.com/packit-service/src/chrony/-/merge_requests/1#note_510963266